pandas/core/indexes/multi.py

```diff
@@ -403,14 +410,19 @@ def _verify_integrity(self, codes: list | None = None, levels: list | None = Non
         # nlevels matches nor that sortorder matches actually sortorder.
         codes = codes or self.codes
         levels = levels or self.levels
+        if levels_to_verify is None:
+            levels_to_verify = list(range(len(levels)))
```

Comment:
Nit: Would be good not to materialize the range since iter and contains should still work.

Reply:
Done